plsc

```diff
@@ -1,256 +0,0 @@
-#!/usr/bin/env python3
```

Comment:
Waarom hernoem je deze files allemaal naar .py?  Ik kan nu namelijk niet zoen wat er is veranderd na de rename.